### PR TITLE
fix: handle custom description for terrasos credibility card section

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: APP-489

Handle custom description for Terrasos credibility card section.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie You can check that the credibility card description has been updated here: https://deploy-preview-2555--terrasos.netlify.app/project/C02-009

**Comment by erikalogie** [comment]
That looks good, can you also move the credibility cards up so that the arrows are in alignment with the paragraph on the right? blob:https://regennetwork.atlassian.net/50819757-ddae-4acf-94c3-48b4181085c6#media-blob-url=true&id=20f4dbce-b530-477a-8d3b-276c4512d9e0&contextId=12161&collection=

**Comment by erikalogie** [comment]
Oh weird, here is the image
image

**Comment by erikalogie** [comment]
It looks good on desktop but on mobile it is overlapping:
Screenshot 2024-11-25 at 12 51 59 PM

File: web-marketplace/src/components/organisms/DetailsSection/DetailsSection.tsx [hunk 1]
```diff
@@ -56,7 +61,19 @@ export const DetailsSection: React.FC<
                 {header.title}
               </Title>
               <Body size="lg" mobileSize="md" maxWidth={718}>
-                <BlockContent content={header.descriptionRaw} />
+                {IS_REGEN ? (
+                  <BlockContent content={header.descriptionRaw} />
+                ) : (
+                  <div>
+                    {_(TERRASOS_PROJECT_DESCRIPTION)}
+                    <Link
```
Comment (blushi) [inline web-marketplace/src/components/organisms/DetailsSection/DetailsSection.tsx:69]: nothing happens when I click on the link
this seems to be related to another component overlapping

Comment (flagrede) [reply web-marketplace/src/components/organisms/DetailsSection/DetailsSection.tsx:74]: Indeed, it's fixed now.

**Comment by flagrede** [comment]
@erikalogie Should be better now.

**Comment by flagrede** [comment]
@blushi I have fixed the long description issue by making the container smaller for the Regen client.

**Comment by erikalogie** [comment]
LGTM!

**Pull request merged**